tmgmt_zanata.ui.inc

```diff
@@ -72,6 +72,26 @@ class TMGMTZanataTranslatorUIController extends TMGMTDefaultTranslatorUIControll
         'Automatically fetch new translations every time cron runs.'),
       '#default_value' => $translator->getSetting('transcheck'),
     );
+    $form['emailnotification'] = array(
+        '#type' => 'checkbox',
+        '#title' => t('Send an email when a job is created'),
+        '#description' => t(
+            'Automatically send an email every time job is created.'),
```

Comment:
The best would be if the checkbox and email field can be grouped in a box that has a description of the type of emails that can be sent, followed by these two options with the current descriptions as they appear now.